src/Components/AddTodo.jsx

```diff
@@ -0,0 +1,72 @@
+import { useState, useEffect } from "react";
+
+function AddTodo({ addTodo, hideAddTodo }) {
+  const [todoInput, setTodoInput] = useState("");
+  const [alarmTime, setAlarmTime] = useState("");
```

Comment:
always use object as much as possible, for multiple input you should not create multiple states.
I think, I was clear last time about this. Do not make me repeat my self again-2 for same stuff @kaushaki-kumari